serving/samples/autoscale-go/README.md

```diff
@@ -112,43 +212,43 @@ kubectl port-forward --namespace knative-monitoring $(kubectl get pods --namespa
 
 ### Other Experiments
 
-1. Maintain 1000 concurrent requests.
+1. Maintain 100 concurrent requests.
```

Comment:
This says maintain still, but the command below runs for 60 second while the previous command does not look to have a timeout.

Same for the two below.

Reply:
Good point.  I've updated to comments to reflect the limited test duration of 60 seconds.